functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.validation/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/ValidationModulesExtFilesTests.java

```diff
@@ -95,7 +93,7 @@ public void checkExtFilesWithoutProjects() {
 
 		validationAction.assertEnabledTrue(wizardAction.getBackBtn());
 
-		validationAction.assertEnabledTrue(wizardAction.getFinishBtn());
+		validationAction.assertEnabledFalse(wizardAction.getFinishBtn());
```

Comment:
6. add one test case to test module ext project name
mainly on: when no ext project, check available project name
create two ext project, ext files wizard check available project names